axelrod/tests/strategies/test_grudger.py

```diff
@@ -128,24 +154,26 @@ class TestSoftGrudger(TestPlayer):
     }
 
     def test_strategy(self):
+        self.maxDiff = None
         # Starts by cooperating.
         self.first_play_test(C)
         # If opponent defects at any point then the player will respond with
         # D, D, D, D, C, C.
-        self.responses_test([C], [C], [C])
-        self.responses_test([D], [C, C], [C, D])
-        self.responses_test([D], [C, C, D], [C, D, C])
-        self.responses_test([D], [C, C, D, D], [C, D, C, C])
-        self.responses_test([D], [C, C, D, D, D], [C, D, C, C, C])
-        self.responses_test([C], [C, C, D, D, D, D], [C, D, C, C, C, C])
-        self.responses_test([C], [C, C, D, D, D, D, C], [C, D, C, C, C, C, C])
-        self.responses_test([D], [C, C, D, D, D, D, C, C],
-                            [C, D, C, C, C, C, C, D])
-        self.responses_test([D], [C, C, D, D, D, D, C, C, D],
-                            [C, D, C, C, C, C, C, D, C])
+        vs_cooperator = [(C, C)] * 30
+        self.versus_test(axl.Cooperator(), expected_actions=vs_cooperator)
```

Comment:
To be in line with tests already written could we stick with `actions=[(C, C)] * 30` instead of `vs_cooperator`. (Similarly elsewhere).